### PR TITLE
Update README.TXT to avoid dead link

### PR DESCRIPTION
**Pull request by dougbu**
- #9

### CODE REVIEW

File: samples/InstagramReceiver/README.TXT [hunk 1]
```diff
@@ -28,8 +28,7 @@ You don't have to be logged in using your Instagram credentials to subscribe or
 using Instagram credentials is so that the server can download media from your Instagram when you post data and a 
 WebHook notification is received.
 
-See http://www.oauthforaspnet.com/providers/instagram/ for information about how to register an application with Instagram 
-and how to authenticate it. 
+See https://github.com/TerribleDev/OwinOAuthProviders for information about the Owin.Security.Providers package used in this sample. Could also depend on just the Owin.Security.Providers.Instagram package.
```
Comment (Eilon) [inline samples/InstagramReceiver/README.TXT]: > Could also depend on just the Owin.Security.Providers.Instagram package.

What does that mean?

Comment (dougbu) [reply samples/InstagramReceiver/README.TXT]: Owin.Security.Providers is a meta-package that brings in Owin.Security.Providers.Instagram and a zillion other Owin security providers. Depend on Owin.Security.Providers.Instagram and you just get the one you need.

Comment (Eilon) [reply samples/InstagramReceiver/README.TXT]: But why would we put that in our own readme? Who will read the comment and why would they act on it?

Comment (Eilon) [reply samples/InstagramReceiver/README.TXT]: I guess we should make the sample/readme show the 1 best recommendation that we have. If we recomment using the Instagram package directly, the sample and readme should say that. If we recommend the megapackage, we should say just that.

**Comment by dougbu** [comment]
5352b81e9b

**Pull request closed without merging**